### PR TITLE
Removed "Ports/Windows port" subdirectory which contained a single file

### PR DESCRIPTION
**Pull request by fujii**
* Renamed "Ports/Windows port/Introduction.md" to "Ports/WindowsPort.md"
* Fixed broken links.

### CODE REVIEW

File: docs/Build & Debug/BuildOptions.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ Tools/Scripts/build-webkit --wpe --debug
 
 ## Building Windows Port
 
-For building WebKit on Windows, see [Windows Port](../Ports/Windows port/Introduction.html).
```
Comment (stwrt) [inline docs/Build & Debug/BuildOptions.md:87]: Adding .html causes the link to fail. Removing .html seems to fix the link.

File: docs/Ports/WindowsPort.md [hunk 2]
```diff
@@ -108,7 +108,7 @@ The build succeeded if you got `WebKit is now built` message. Run your `MiniBrow
 WebKitBuild/Release/bin64/MiniBrowser.exe
 ```
 
-You can run programs under a debugger with [this instruction](../../Build & Debug/DebuggingWithVS.html).
```
Comment (stwrt) [inline docs/Ports/WindowsPort.md:111]: Adding .html causes the link to fail. Removing .html seems to fix the link.

**Review by stwrt — CHANGES_REQUESTED** [review]
Two very minor things. The .html seems to be breaking the links when testing this locally. Does it work for you?

File: docs/Build & Debug/BuildOptions.md (hunk 1 above)
Comment (fujii) [reply docs/Build & Debug/BuildOptions.md:87]: It works fine with `.html`, but doesn't work without `.html` on my PC.
The following line also appends `.html`. 
https://github.com/WebKit/Documentation/blob/main/docs/Getting%20Started/ContributingCode.md?plain=1#L15
The link on the web site is broken.
https://docs.webkit.org/Getting%20Started/ContributingCode.html
MkDocs version issue?

Comment (fujii) [reply docs/Build & Debug/BuildOptions.md:87]: The document says we have to use relative paths and `.md` suffix.
https://www.mkdocs.org/user-guide/writing-your-docs/#linking-to-pages

Comment (fujii) [reply docs/Build & Debug/BuildOptions.md:87]: Fixed.

File: docs/Ports/WindowsPort.md (hunk 2 above)
Comment (fujii) [reply docs/Ports/WindowsPort.md:111]: Fixed.

**Review by stwrt — APPROVED** [review]
Tested this again after upgrading mkdocs and it worked great. Must have been an issue on my end. Approving.

**Comment by stwrt** [comment]
Looks like there were some merge conflicts that need to be resolved.

**Pull request merged**